app/admin/sponsor.rb

```diff
@@ -28,8 +28,8 @@
     column :start_date
     column :request_fulfilled
     column :sponsor_type
-    column :country do |sponsor|
-      ISO3166::Country.search(sponsor.country)
+    column :country do |_sponsor|
+      "#{t _sponsor.country, locale: :ar} (#{ISO3166::Country[_sponsor.country]})"
```

Comment:
Valid point, but both you and I are guessing. I'm OK to leave it up to the client.